src/main/java/net/sf/jabref/sql/database/MySQL.java

```diff
@@ -0,0 +1,129 @@
+package net.sf.jabref.sql.database;
+
+import net.sf.jabref.model.entry.IdGenerator;
+import net.sf.jabref.sql.DBStrings;
+import net.sf.jabref.sql.Database;
+import net.sf.jabref.sql.SQLUtil;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class MySQL implements Database {
+
+    public static final String DRIVER = "com.mysql.jdbc.Driver";
+
+    private void loadDriver() throws ClassNotFoundException, IllegalAccessException, InstantiationException {
+        Class.forName(DRIVER).newInstance();
+    }
+
+    @Override
+    public Connection connect(String url, String username, String password)
+            throws IllegalAccessException, InstantiationException, ClassNotFoundException, SQLException {
+        loadDriver();
+
+        return DriverManager.getConnection(url, username, password);
+    }
+
+    @Override
+    public String getReadColumnNamesQuery() {
+        return "SHOW columns FROM entries;";
+    }
+
+    @Override
+    public String getCreateTableSQL(Table table) {
+        switch (table) {
+
+        case JABREF_DATABASE:
```

Comment:
At this point I would suggest better creating a method for each case  
e.g. createhabRefDatabase 
That would better follow the principle of a single purpose method and additionally reduces the length of the method. Furthermore better readability